Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -2562,8 +2562,9 @@ def _http_request(self, method, url_suffix, full_url=None, headers=None,
                         error_entry = res.json()
                         err_msg += '\n{}'.format(json.dumps(error_entry))
                         raise DemistoException(err_msg)
-                    except ValueError as exception:
-                        raise DemistoException(err_msg, exception)
+                    except ValueError:
+                        err_msg += '\n{}'.format(res.text)
+                        raise DemistoException(err_msg)
```

Comment:
```suggestion
                        raise DemistoException(err_msg, exception)
```

Reply:
Why is this necessary? Why should we care about the `ValueError`?
Don't you think it's blinding?

Reply:
what do you mean by blinding? it just prints it to the logs

Reply:
I mean that the value Error is not really the error we encountered, it's a consequence of the real error.

In the UI it looks like this
![image](https://user-images.githubusercontent.com/41257953/77628585-ea77ad80-6f50-11ea-8336-8a892ffa221a.png)

rather then like this:
![image](https://user-images.githubusercontent.com/41257953/77628687-0d09c680-6f51-11ea-9cd2-a9ed9e1b1d7e.png)